packages/commons/src/rate-limiter/redisRateLimiter.ts

```diff
@@ -25,21 +26,54 @@ export async function initRedisRateLimiter(config: {
   redisPort: number;
   timeout: number;
 }): Promise<RateLimiter> {
-  const redisClient = await createRedisClient({
+  const redisClient = createRedisClient({
+    // legacyMode: true, // Use legacy mode for compatibility with rate-limiter-flexible
+    disableOfflineQueue: true, // disable the offline queue to make requests fail immediately if Redis is down
+    // so that the rate limiter fallbacks to the insurance limiter
```

Comment:
```suggestion
    disableOfflineQueue: true,
    /* ^ disable the offline queue to make requests fail immediately if Redis is down
    so that the rate limiter falls back to the insurance limiter */
```